### PR TITLE
Fix Fishing Rod

### PR DESCRIPTION
**Pull request by KennFatt**
- Change Fishing Rod category from **Item** to **Tool**
- Damageable : 65 Durability _**based minecraft wiki**_
- Set max count to 1 (_as default tool_)
- Fix "**Unknown**" to "**Fishing Rod**"

### CODE REVIEW

File: src/pocketmine/item/Tool.php [hunk 1]
```diff
@@ -146,6 +147,6 @@ public function isShears(){
 	}
 
 	public function isTool(){
-		return ($this->id === self::FLINT_STEEL or $this->id === self::SHEARS or $this->id === self::BOW or $this->isPickaxe() !== false or $this->isAxe() !== false or $this->isShovel() !== false or $this->isSword() !== false);
+		return ($this->id == self::FISHING_ROD or $this->id === self::FLINT_STEEL or $this->id === self::SHEARS or $this->id === self::BOW or $this->isPickaxe() !== false or $this->isAxe() !== false or $this->isShovel() !== false or $this->isSword() !== false);
```
Comment (corytortoise) [inline src/pocketmine/item/Tool.php]: I think you forgot an = in this line.

**Comment by jasonwynn10** [comment]
That's a lot of commits. This isn't going to get merged

**Comment by JackNoordhuis** [comment]
> __Try to stick to one change per commit.__ This ensures that if you create a PR with several changes, we can decide which ones we wish to include and which ones not to include.

Please read the [Contribution guidelines](https://github.com/pmmp/PocketMine-MP/blob/master/CONTRIBUTING.md) before making pr's.

**Comment by KennFatt** [comment]
@JackNoordhuis @jasonwynn10 idk how it can become lot of commit, my commit its only 3 :( should i close it?

**Comment by jasonwynn10** [comment]
rebase onto the master branch

**Comment by jasonwynn10** [comment]
Yes, then learn how to use git.

**Comment by dktapps** [comment]
The diff is heavily polluted. Please see comments above.

**Pull request closed without merging**